src/services/content.ts

```diff
@@ -300,6 +300,12 @@ const handleBatchAnnouncement = (dispatch: Dispatch) => (
           })
         );
       } else if (isBroadcastAnnouncement(announcement)) {
+        let publishedDate;
+        if (announcement.createdAt > (2 ^ 53) - 1) {
```

Comment:
```suggestion
        if (announcement.createdAt > Number.MAX_SAFE_INTEGER) {
```
This is an actual value: Number.MAX_SAFE_INTEGER and should be used instead of being calculated: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER